plugins/hypervisors/xenserver/src/main/java/com/cloud/hypervisor/xenserver/resource/CitrixHelper.java

```diff
@@ -44,4 +45,14 @@ public static String getPVbootloaderArgs(String guestOS) {
         }
         return "";
     }
+
+    public static String getSRNameLabel(final String poolUuid,
+                                        final Storage.StoragePoolType poolType,
+                                        final String poolPath) {
+        if (Storage.StoragePoolType.PreSetup.equals(poolType) &&
+                !poolPath.contains(poolUuid)) {
```

Comment:
aren't there any presetup scenarios where the path could contain the uuid?

Reply:
@DaanHoogland For presetup pools, UUID in DB is set as the path of the pool (without `/`)
But in ACP it was using real UUIDs as reported in the linked issue.
So my change is just to use path (without `/`) when there is a mismatch between path and UUID and the pool is of type presetup